.github/pull_request_templae.md

```diff
@@ -0,0 +1,44 @@
+[Review Guidance](https://genesisglobal.atlassian.net/wiki/spaces/ON/pages/2682257431/Platform+Code+Review+Process)
```

Comment:
is this link accessible to anyone?

Reply:
good spot, no it's not, wasn't migrated over to notion

Reply:
it was actually migrated to notion, hard to find though

Reply:
worth considering if we want to make it public in case someone outside our org wanted to contribute